src/notification/notification.service.ts

```
@@ -0,0 +1,199 @@
+          itineraryId: reminder.itineraryId,
+          recipient: reminder.email,
+          recipientName: reminder.recipientName,
+          tripName: reminder.tripName,
+          reminderOption: reminder.reminderOption as
+            | 'TEN_MINUTES_BEFORE'
+            | 'ONE_HOUR_BEFORE'
+            | 'ONE_DAY_BEFORE',
+          startDate: reminder.startDate.toISOString(),
+        })
+      } catch (error) {
+        continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)